.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Should we include `.vscode` to `.gitignore` file?

Reply:
I think it's okay to leave it there...We do want the eslint settings to be in the repo, don't we? Or it's some other settings that you're worried about?

Reply:
I just don't want to override the setting at some point. Otherwise, it's fine to keep it.

Reply:
Ah I see :) Not sure if Martin wants to make anymore changes in this, so I'm just leaving it there for now :p